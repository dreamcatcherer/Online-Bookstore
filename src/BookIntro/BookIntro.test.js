import React from "react";
import { render } from "@testing-library/react";
import BookIntroduction from "./BookIntro.jsx";

it("renders the BookIntroduction component correctly", () => {
  const introduction="There comes a point in a toddler’s life when going in one’s diaper is only one possible option, and the question must be raised: \"Should I go in my potty?\". With pitch-perfect humor and pacing, Leslie Patricelli follows the inner dialogue and hilarious actions of everyone’s favorite Baby, winding up with an over-the-top look of surprise and delight that will have both parents and offspring laughing out loud--\"I did it!\"";

  const { container } = render(<BookIntroduction intro={introduction} />);
  expect(container).toMatchSnapshot();
});
