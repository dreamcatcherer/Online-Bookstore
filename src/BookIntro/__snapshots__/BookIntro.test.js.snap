// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders the BookIntroduction component correctly 1`] = `
<div>
  <div
    class="introductionStyle"
  >
    <h2>
      Introduction
    </h2>
  </div>
</div>
`;
